Dockerfile

```diff
@@ -71,4 +71,4 @@ RUN mkdir /bzt-configs \
   && rm -rf /tmp/*
 
 WORKDIR /bzt-configs
-CMD bzt -l /tmp/artifacts/bzt.log /bzt-configs/*.yml
+ENTRYPOINT ["sh", "-c", "bzt /bzt-configs/*.yml -l /tmp/artifacts/bzt.log \"$@\"", "ignored"]
```

Comment:
Hi,
Thanks for detailed explanation. I like the solution 2, because it looks backwards compatible and allows manual option override. Or solution 3, also fine. I don't like implicit "magic" of `*.yml *.json` inside docker, it's not obvious for users.